app/controllers/admin/users_controller.rb

```diff
@@ -13,13 +13,13 @@ def new
     end
 
     def create
-      @user = User.new(user_params)
-      @user.confirmed_at = Time.now
+      @user = User.new(user_params.merge(confirmed_at: Time.zone.now))
+      @user.password = "Password123!@#"
       if @user.save
         redirect_to admin_users_path, notice: "User added"
-        WelcomeMailer.welcome_email(@user).deliver
+        @user.invite(@user)
```

Comment:
you do not need to pass the `user` argument as it is an instance method and the user should be available as self in the method.